gpii/node_modules/processReporter/src/processesBridge.js

```diff
@@ -0,0 +1,236 @@
+/*!
+GPII Process Reporter processes bridge -- gpii.processes.
+
+Copyright 2014 Inclusive Design Research Centre, OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+/*global require */
+
+"use strict";
+
+var fluid = require("infusion");
+var gpii = fluid.registerNamespace("gpii");
+
+gpii.processes = fluid.registerNamespace("gpii.processes");
+
+fluid.defaults("gpii.processes", {
```

Comment:
Since this functionality is self-contained in universal, it should have self-contained tests here also. Rather than "binding blind" to the grade gpii.processes.native, I suggest using the following pattern of applying the Infusion contextAwareness feature: https://github.com/GPII/universal/pull/515/files#diff-72d357ed506d19873d122363254ccf51R28

You can then supply a mock grade in your tests which responds to the context grade "gpii.contexts.test" which is always defined within a test case - https://github.com/GPII/universal/blob/master/index.js#L43